static/app/views/app/index.tsx

```diff
@@ -228,14 +229,15 @@ function App({children, params}: Props) {
 
   // Used to restore focus to the container after closing the modal
   const mainContainerRef = useRef<HTMLDivElement>(null);
-  const handleModalClose = useCallback(() => mainContainerRef.current?.focus?.(), []);
+  const focusMainContainer = useCallback(() => mainContainerRef.current?.focus?.(), []);
 
   return (
     <Profiler id="App" onRender={onRenderCallback}>
       <OrganizationContextProvider>
         <AsyncSDKIntegrationContextProvider>
           <MainContainer tabIndex={-1} ref={mainContainerRef}>
-            <GlobalModal onClose={handleModalClose} />
+            <GlobalModal onClose={focusMainContainer} />
+            <GlobalDrawer onClose={focusMainContainer} />
```

Comment:
Switching this to a context provider will mean the GlobalDrawer wraps everything.